src/build/plan.rs

```diff
@@ -352,7 +358,19 @@ impl JobQueue {
                 .map(|x| x.into_string().unwrap())
                 .collect();
 
-            args.insert(0, job.get_program().clone().into_string().unwrap());
```

Comment:
Why was this line split into `let program = ...` and `args.insert(0, program);` with progress update in between?

Reply:
It's a remnant from when I sent the `program` also as part of the percentage message in a previous iteration. I'll restore it as it was.

I want to do a git rebase once we've iterated this enough to make the commit log more sensible.